### DISCUSSION TITLE
issue with organization invitation

### DISCUSSION THREAD

**Discussion by SunWon-Park** [community/community#138244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi GitHub Support,
I am trying to receive an invitation from the organization "U-Mpay-Service," but the organization owner is unable to send me an invite because I am "opted-out" of receiving invitations.
Could you kindly remove me from the opt-out list so I can receive the invitation?
Thank you for your assistance.

**Comment by ebndev** [comment]
Hi @SunWon-Park,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
